lms/settings.py

```diff
@@ -89,10 +89,18 @@
     }
 }
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

Comment:
I got an error when try to reset password.
![image](https://user-images.githubusercontent.com/17034253/139937557-02e3d8a0-95a8-472c-ace4-9bd5c0f92ef5.png)

Reply:
I can't reproduce this error (( Added @csrf_exempt to the view function